system_test/common/aest_community_fork_SUITE.erl

```diff
@@ -0,0 +1,166 @@
+-module(aest_community_fork_SUITE).
+
+%=== EXPORTS ===================================================================
+
+% Common Test exports
+-export([all/0]).
+-export([init_per_suite/1]).
+-export([init_per_testcase/2]).
+-export([end_per_testcase/2]).
+-export([end_per_suite/1]).
+
+% Test cases
+-export([fork_chain/1]).
+
+-import(aest_nodes, [
+    setup_nodes/2,
+    start_node/2,
+    stop_node/3,
+    wait_for_value/4,
+    wait_for_startup/3,
+    get_block/2,
+    get_top/1
+]).
+
+%=== INCLUDES ==================================================================
+
+-include_lib("stdlib/include/assert.hrl").
+
+%=== MACROS ====================================================================
+
+-define(MINING_TIMEOUT,   3000).
+
+-define(SIGNALLING_START_HEIGHT, 5).
+-define(SIGNALLING_END_HEIGHT, 15).
+-define(SIGNALLING_BLOCK_COUNT, 9).
+-define(FORK_HEIGHT, 20).
+-define(INFO_FIELD, 1234).
+-define(VERSION, 5).
+
+-define(FORK,
+        #{fork =>
+              #{signalling_start_height => ?SIGNALLING_START_HEIGHT,
+                signalling_end_height => ?SIGNALLING_END_HEIGHT,
+                signalling_block_count => ?SIGNALLING_BLOCK_COUNT,
+                fork_height => ?FORK_HEIGHT,
+                info_field => ?INFO_FIELD,
+                version => ?VERSION}}).
+
+-define(MINER_NODE1,
+        #{name            => miner_node1,
+          peers           => [miner_node2, miner_node3],
+          fork_management => ?FORK,
+          mine_rate       => 1000,
+          backend         => aest_docker,
+          source          => {pull, "aeternity/aeternity:local"}}).
+
+-define(MINER_NODE2,
+        #{name            => miner_node2,
+          peers           => [miner_node1, miner_node3],
+          fork_management => ?FORK,
+          mine_rate       => 1000,
+          backend         => aest_docker,
+          source          => {pull, "aeternity/aeternity:local"}}).
+
+%% This node is not configured for fork.
+-define(MINER_NODE3,
+        #{name            => miner_node3,
+          peers           => [miner_node1, miner_node2],
+          mine_rate       => 1000000, %% The node shouldn't mine so it doesn't produce blocks with wrong info field.
```

Comment:
This may affect block validation. Is there way to properly disable mining autostart?

Reply:
aest_db_SUITE seems to have `autostart: {{mining.autostart}}` in the config. Would that help?

Reply:
Fixed.